content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -55,22 +55,65 @@ The underlying SQL looks like this:
 ```sql
 CREATE TABLE "Category" (
     id SERIAL PRIMARY KEY
+    name TEXT NOT NULL
 );
+
 CREATE TABLE "Post" (
     id SERIAL PRIMARY KEY
+    title TEXT NOT NULL
 );
--- Relation table + indexes -------------------------------------------------------
+
+-- Relation table  -------------------------------------------------------
+CREATE TABLE "CategoriesOnPosts" (
+    "categoryId" INTEGER NOT NULL,
+    "postId" INTEGER NOT NULL,
+    "assignedBy" TEXT NOT NULL,    
+    "assignedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY ("categoryId") REFERENCES "Category"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    FOREIGN KEY ("postId") REFERENCES "Post"("id") ON DELETE RESTRICT ON UPDATE CASCADE,
+    CONSTRAINT "CategoriesOnPosts_pkey" PRIMARY KEY ("postId", "categoryId")
+);
+```
+
+<details><summary>Expand for a migration SQL that is generated from this schema</summary>
+
+```sql
+-- CreateTable
```

Comment:
This unfortunately confuses our renderer:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/183673/162049245-50618964-5a6e-4dcd-85b9-87c5be5c82c5.png">
@tanberry @nikolasburk  Do we have a way to fix/avoid this?

Reply:
Odd that when you look at Preview deploy of this page, it results in only one hyphen (- CreateTable) in this first instance, but in subsequent instances we have two hyphens (-- CreateTable).

Is this what you are commenting on, @janpio ?

Reply:
The correct content is double hyphen.
Our renderer understands the first one as an indication to format the content as a deletion in a diff (see the red background color on the line in the screenshot). That should not happen.